package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wemo-client",
-  "version": "0.6.2",
+  "version": "0.6.6",
```

Comment:
Please leave the version number untouched. I'll bump it when publishing a new release.